hubapi/README.MD

```diff
@@ -0,0 +1,75 @@
+`HubAPI`
+The HubAPI supports operations for listing jina hub images
+and pushing images to the hub. These are authorised using the
+hub authorizer. We use MongoDB for storing and retrieving
+information related to hub. This document elaborates on the functioning
+of each of these APIs
+1. `Hub List` API
```

Comment:
`hub list` since its a command (all commands should be exactly as you'd write on command line)